src/main/java/ru/testtask/dto/UserDTO.java

```diff
@@ -5,12 +5,16 @@
 import lombok.NoArgsConstructor;
 import ru.testtask.model.Role;
 
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
 import java.util.Set;
 
 @Data
 @NoArgsConstructor
 public class UserDTO {
```

Comment:
In my opinion it is better to rename to `CreateUpdateUserDTO`. This can extends `ViewUserDTO`